runway/commands/env.py

```diff
@@ -336,28 +336,40 @@ def run(self, deployments=None, command='plan'):  # noqa pylint: disable=too-man
                 deployments_to_run = self.select_deployment_to_run(
                     deployments
                 )
-        for deployment in deployments_to_run:
+
+        LOGGER.info("Found %d deployment(s)", len(deployments_to_run))
+        for i, deployment in enumerate(deployments_to_run):
+            LOGGER.info("")
+            LOGGER.info("======= Processing deployment %d ===========================", i)
+
             if deployment.get('regions'):
                 if deployment.get('env_vars'):
-                    context.env_vars = merge_dicts(
-                        context.env_vars,
-                        get_deployment_env_vars(context.env_name,
-                                                deployment['env_vars'],
-                                                self.env_root)
-                    )
+                    deployment_env_vars = get_deployment_env_vars(context.env_name,
+                                                              deployment['env_vars'],
+                                                              self.env_root)
+                    LOGGER.info("env_vars for this deployment: %s", str(deployment_env_vars))
+                    context.env_vars = merge_dicts(context.env_vars, deployment_env_vars)
+
+                LOGGER.info("")
+                LOGGER.info("Attempting to deploy '%s' to region(s): %s",
+                            context.env_name,
+                            ", ".join(deployment['regions']))
+
                 for region in deployment['regions']:
+                    LOGGER.info("")
+                    LOGGER.info("======= Processing region %s ===========================", region)
+
                     context.env_region = region
                     context.env_vars = merge_dicts(
                         context.env_vars,
                         {'AWS_DEFAULT_REGION': context.env_region,
                          'AWS_REGION': context.env_region}
                     )
                     if deployment.get('assume-role'):
-                        pre_deploy_assume_role(deployment['assume-role'],
-                                               context)
-                    if deployment.get('account-id') or (
-                            deployment.get('account-alias')):
+                        pre_deploy_assume_role(deployment['assume-role'], context)
```

Comment:
these next few are not changes per se, just removing newlines that aren't need anymore, and were just confusing